modules/op-release-notes-1-15.adoc

```diff
@@ -460,7 +460,7 @@ With this update, {pipelines-title} General Availability (GA) 1.15.1 is availabl
 [id="new-features-1-15-1_{context}"]
 === New features
 
-* Before this update, in the `TektonConfig` CR, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supported only the `bool` value type. With this update, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supports both the `bool` and `string` value types.
+* Before this update, in the `TektonConfig` CR, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supported only the `bool` value type. With this update, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supports only the `string` value type; the valid values for this spec are `"true"` and `"false"`.
```

Comment:
With the update, `chain.artifacts.pipelinerun.enable-deep-inspection` actually supports both bool AND string, with the valid values for the spec `"true"` and `"false"`.
Let me explain the purpose of the change. We want to allow the user to use both `true`/`false` as well as `"true"`/`"false"` and not limit it to specific type. @rupalibehera (who worked and the change) and @jkandasa, can you please confirm?

Reply:
No, this was the previous version of the RN and it's not how things actually work. In actual fact the bool type is NOT supported and one HAS to use `"true"`/`"false"` string values.

Reply:
@lahinson thanks for the suggestion but I'll keep the "spec" because it refers to the `spec:` section of a YAML manifest; we use this term throughout the doc.